wolfssl/wolfcrypt/settings.h

```diff
@@ -2504,7 +2504,6 @@ extern void uITRON4_free(void *p) ;
     #define WOLFSSL_DH_CONST
     #define WOLFSSL_HAVE_MAX
     #define NO_WRITEV
-    #define NO_STDLIB_ISASCII
```

Comment:
It just occurred to me, that `HAVE_ISASCII` is stored in `config.h` which is optional. Maybe this define should stay as is?